src/manifest.rs

```diff
@@ -212,3 +212,28 @@ fn check_crate_type(path: &Path) -> Result<(), Error> {
       "crate-type must be cdylib to compile to wasm32-unknown-unknown. Add the following to your Cargo.toml file:\n\n[lib]\ncrate-type = [\"cdylib\"]"
     )
 }
+
+/// Get the version of `wasm-bindgen` specified as a dependency.
+pub fn get_wasm_bindgen_version(path: &Path) -> Result<String, Error> {
+    if let Some(deps) = read_cargo_toml(path)?.dependencies {
+        match deps.get("wasm_bindgen") {
```

Comment:
Is this correct? I thought it would be "wasm-bindgen". This line is giving me problems locally.

Reply:
See https://github.com/rustwasm/wasm-pack/pull/267